frontend/packages/dev-console/src/components/topology/shapes/PodSet.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { get } from 'lodash';
+import {
+  PodStatus,
+  calculateRadius,
+  getDataBasedOnCurrentAndPreviousRC,
+  rollback,
+} from '@console/shared';
+import { k8sUpdate } from '@console/internal/module/k8s';
+import { DeploymentConfigModel } from '@console/internal/models';
+import { DonutStatusData } from '../topology-types';
+
+interface PodSetProps {
+  size: number;
+  data: DonutStatusData;
+}
+
+const calculateInnerPodStatusRadius = (
+  outerPodStatusInnerRadius: number,
+  outerPodStatusWidth: number,
+): { innerPodStatusOuterRadius: number; innerPodStatusInnerRadius: number } => {
+  const innerPodStatusWidth = outerPodStatusWidth * 0.6;
+  const spaceBwOuterAndInnerPodStatus = 3;
+  const innerPodStatusOuterRadius = outerPodStatusInnerRadius - spaceBwOuterAndInnerPodStatus;
+  const innerPodStatusInnerRadius = innerPodStatusOuterRadius - innerPodStatusWidth;
+
+  return { innerPodStatusOuterRadius, innerPodStatusInnerRadius };
+};
+
+const PodSet: React.FC<PodSetProps> = ({ size, data }) => {
+  const { podStatusOuterRadius, podStatusInnerRadius, podStatusStrokeWidth } = calculateRadius(
+    size,
+  );
+  const { innerPodStatusOuterRadius, innerPodStatusInnerRadius } = calculateInnerPodStatusRadius(
+    podStatusInnerRadius,
+    podStatusStrokeWidth,
+  );
+  const currentRCPhase = get(data.current, 'metadata.name');
```

Comment:
It's better to use `_,get` to be explicit about using a lodash function and it will be consistent with how we use lodash everywhere else as well. We do have ES6 module plugin for lodash in webpack setup that does the tree shaking for lodash so that we do not load the whole library.

Reply:
fixed in https://github.com/openshift/console/pull/2456/commits/d069d28b40bb578d9a26810bb61269647f5d241b